restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+= Restcomm Release Notes
+
+These notes group both Community and Telscale changes in a single document.
+Changes marked with (R) sign, means those changes are introduced for the respective
+Telscale commercial product.
+
+//add release-notes with newer on top
+== 8.4.0 version
+=== Additions 
+// New features (whether major or minor) go here
+* Clients passwords are now hashed in DB.
+* Profiles allow arbitrary properties to be saved/retrieved.
+* Added configurable SBC mode.
+
+=== Removals
```

Comment:
now that i see the content, i think "Breaking Changes" might make more sense...

Reply:
yep, i think even for end-user the distiction of bugfixes/newFeatures/breakingchanges is still clear and valuable, we can find diff names so we dont use the SemVer references, but i think is better than  these additions/removals/changes. to me those are not clear, i had hard time finding proper category

Reply:
Changed to `New Features`, `Breaking changes`, `Bug Fixes`